KsApi/models/Checkout.swift

```diff
@@ -2,22 +2,17 @@ import Foundation
 
 public struct Checkout: Decodable {
   public var state: State
+  public var backing: Backing
 
   public enum State: String, Decodable, CaseIterable {
     case authorizing = "AUTHORIZING"
     case verifying = "VERIFYING"
     case successful = "SUCCESSFUL"
     case failed = "FAILED"
   }
-}
-
-extension Checkout {
-  enum CodingKeys: String, CodingKey {
-    case state
-  }
 
-  public init(from decoder: Decoder) throws {
-    let values = try decoder.container(keyedBy: CodingKeys.self)
-    self.state = try values.decode(State.self, forKey: .state)
+  public struct Backing: Decodable {
```

Comment:
Wondering why not just add these two properties to our existing `Backing` object 🤔 are you thinking that it would be too much work to conform `Backing` to `Swift.Decodable`? I'm just worried about having multiple `Backing` objects in different scopes.

Reply:
We could do that, I guess we'd just have to query for a few things here in `Backing` that we don't really need in this scope?

Also, I think it's ok to have something like `Checkout.Backing` and `Backing`. Being nested within `Checkout` makes it fairly clear that it's not the same thing.

But open to the idea.

Reply:
Thinking let's not do this right now?